lib/jazzy/config.rb

```diff
@@ -226,6 +226,19 @@ def expand_path(path)
                    'comments.',
       default: false
 
+    config_attr :lint_report,
+      description: ['When provided, a tab-separated table of results will be written',
+                    'to the file. If a relative path is provided, the path will be ',
+                    'considered relative to the current working directory.',
```

Comment:
This isn’t quite correct. Because you (rightly) use `expand_path`, it's relative to working dir if specified on the command line, relative to the config file's parent if specified in a config file.

Since this is standard for all path-based options and is explained in the online help, no need to explain it here.